site/src/pages/TerminalPage/TerminalPage.tsx

```diff
@@ -28,18 +29,16 @@ import {
   LoadedScriptsAlert,
   LoadingScriptsAlert,
 } from "./TerminalAlerts";
+import { useQuery } from "@tanstack/react-query";
+import { deploymentConfig } from "api/queries/deployment";
 
 export const Language = {
   workspaceErrorMessagePrefix: "Unable to fetch workspace: ",
   workspaceAgentErrorMessagePrefix: "Unable to fetch workspace agent: ",
   websocketErrorMessagePrefix: "WebSocket failed: ",
 };
 
-type TerminalPageProps = React.PropsWithChildren<{
-  renderer: "webgl" | "dom";
-}>;
-
-const TerminalPage: FC<TerminalPageProps> = ({ renderer }) => {
+const TerminalPage: FC = () => {
```

Comment:
I think we are getting a test failure because the test looks for HTML elements so it relies on the `dom` renderer being used (see `webTerminal.spec.ts`).  Maybe we can start the e2e tests with `dom` set as the renderer?  Looks like we would do that in `playwright.config.ts` if I understand correctly.